msteams-platform/bots/design/bots.md

```diff
@@ -99,13 +119,27 @@ Add a tab that educates users about how to communicate with your bot. You can pr
 ![Providing help](~/assets/images/framework/framework_bots_tbot-help.png)
 
 > [!TIP]
-> Embedding parts of your site in a tab will help someone maintain the context of a conversation as they use your service. It removes the need to launch your service in a browser and switch back and forth between apps.
+> Embedding parts of your site in a tab will help users maintain the context of a conversation as they use your service. It removes the need to launch your service in a browser and switch back and forth between apps.
 
 ---
 
-## Best practices
+## Bots in channels
+
+Invoking a bot in a channel can be accomplished by `@mention`. Bot dialog should be unique in channels and groups vs. one-to-one scenarios and it's generally a good idea to consider separate approaches. This is especially true in the following cases:
+
+### Sensitive data sent by a bot
+
+While the users in a team can be known to the service, the actual user roles cannot. This means that, for example, in an education scenario involving bullying, parent and student contact information wouldn't be shared in a team setting. Instead the bot’s message might be,“Two bullying incidents occurred today" along with a button to show details.
+
+Launching details in a web page, or a task module can prompt for user credentials or query against an index for user roles paired with AAD accounts to verify the user. Since both of these options are a private view of the data there can be no data leakage. If the same data were sent in a 1:1 chat between a user and the bot the data is only visible to the user in that context and is therefore safe to fully display in the bot message. Taking users from a channel to a 1:1 chat should be avoided however as that forced navigation is highly disruptive.
```

Comment:
Hi @EmilyyC . Great overall! but I think this paragraph needs clarification. What is meant by "private view" and are we stating that there won't be data leakage or there can't be data leakage? Thank you!

Reply:
it just meant since website or task module are only available to that one particular user, there wont be data leakage. we can also do this In 1:1 chat but that's not a good experience for users since we are taking them out of channel. 

Note: Ojasvi is working on a feature that allow different users to see different content in the card. it's still to call out that developers should be mindful about the sensitive data but our recommendation would be use the card directly instead of web or task module